packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -76,16 +76,21 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
               x: points.chartData![0].horizontalBarChartdata!.y - datapoint!,
               y: points.chartData![0].horizontalBarChartdata!.y,
             },
-            color: palette.neutralTertiaryAlt,
+            color: palette.neutralLight,
           };
 
           const chartDataText = this._getChartDataText(points!);
           const bars = this._createBars(points!, palette);
           const keyVal = this._uniqLineText + '_' + index;
+          const classNames = getClassNames(this.props.styles!, {
+            theme: this.props.theme!,
+            width: this.props.width,
+            showTriangle: !!points!.chartData![0].data,
+          });
 
           return (
-            <div key={index} className={this._classNames.items}>
```

Comment:
what does this change do

Reply:
The margin on class `items` changes depending on the prop `showTriangle` which is specific to each item. `this._classNames` doesn't include that prop, so `getClassNames` function is called for each item with the prop to get the appropriate margin